lib/utils/computed-properties.js

```diff
@@ -38,32 +38,38 @@ function isComputedPropertyBodyArg(node) {
  * @returns {ASTNode} function body of computed property
  */
 function getComputedPropertyFunctionBody(node) {
```

Comment:
Can you add some unit tests for the new functionality in this function?

Reply:
I tried to add unit tests for this, but it relies on using `node.parent` which didn't get populated when I ran `babelEslint.parse(code)`.